src/main/java/com/sourcegraph/cody/chat/CodeEditorFactory.java

```diff
@@ -2,7 +2,5 @@
 
 public class CodeEditorFactory {
 
-  public static final int spaceBetweenButtons = 5;
-
   public static volatile String lastCopiedText = null;
```

Comment:
It looks like since we have moved to webview we do not set this value. Hence, we do not log the paste event.

Or at any chance, does the agent handle the paste event? If it does not, we probably need to send the copied text via a notification to the client to track it in here.

Reply:
https://linear.app/sourcegraph/issue/CODY-4121/fix-keydownpaste-telemetry-events-logging